tests/components/ozw/conftest.py

```diff
@@ -87,6 +111,50 @@ async def light_msg_fixture(hass):
     return message
 
 
+@pytest.fixture(name="light_no_rgb_msg")
+async def light_no_rgb_msg_fixture(hass):
+    """Return a mock MQTT msg with a light actuator message."""
+    light_json = json.loads(
+        await hass.async_add_executor_job(load_fixture, "ozw/light_no_rgb.json")
+    )
+    message = MQTTMessage(topic=light_json["topic"], payload=light_json["payload"])
+    message.encode()
+    return message
+
+
+@pytest.fixture(name="light_no_ww_msg")
+async def light_no_ww_msg_fixture(hass):
+    """Return a mock MQTT msg with a light actuator message."""
+    light_json = json.loads(
+        await hass.async_add_executor_job(load_fixture, "ozw/light.json")
+    )
+    message = MQTTMessage(topic=light_json["topic"], payload=light_json["payload"])
+    message.encode()
+    return message
+
+
+@pytest.fixture(name="light_no_cw_msg")
+async def light_no_cw_msg_fixture(hass):
+    """Return a mock MQTT msg with a light actuator message."""
+    light_json = json.loads(
+        await hass.async_add_executor_job(load_fixture, "ozw/light.json")
+    )
+    message = MQTTMessage(topic=light_json["topic"], payload=light_json["payload"])
+    message.encode()
+    return message
+
+
+@pytest.fixture(name="light_wc_msg")
+async def light_wc_msg_fixture(hass):
+    """Return a mock MQTT msg with a light actuator message."""
+    light_json = json.loads(
+        await hass.async_add_executor_job(load_fixture, "ozw/light.json")
```

Comment:
These three latest fixtures all use the same json file.

Reply:
Correct, I modified the reported color channels to report either no CW or WW, etc. They're all based on your original bulb dump. Figured making a new json with the exact same content would be wasteful.

Reply:
If the fixture is the same we don't need to create more then one.

Reply:
We're not using these fixtures either.

Reply:
They're at the end of the `test_light.py`

Reply:
No, we're not using these fixtures anywhere.

Reply:
I think I understand now, I'll clean them up.